server/csharp/TicketHub/Controllers/TicketController.cs

```diff
@@ -204,13 +206,50 @@ public struct PolicyResult
         public UCASTNode? Conditions;
     }
 
-    private async Task<UCASTNode?> getConditions(HttpContext context, string path, object input)
+    public struct CompileUCASTResult

```

Comment:
This type is a quick hack to allow `Newtonsoft.Json` to properly unpack the `/v1/compile` result into useful types. When column masking is added, this type will likely be extended to cover whatever extra fields are needed.